src/controls/QskProgressBarSkinlet.cpp

```diff
@@ -124,8 +124,6 @@ QSGNode* QskProgressBarSkinlet::updateBarNode(
     if ( !gradient.isVisible() )
         return nullptr;
 
-    gradient.setOrientation( bar->orientation() );
-
```

Comment:
This line was made for a situation, where the gradient is in direction of the progress ( f.e cold/hot -> blue/red ) without having to specify it twice using the placement bits. One of the situations, where we would need something like a QskGradient::NoOrientation or having QskColorStops only.

I recently played with extending the QskGradient::Orientation enum, but was not yet happy with all consequences in other places. Need to think more about it ...

Reply:
I can see the need for this and a new enum. Will put this PR on hold, as it is not that easy...